apps/bin/common.sh

```diff
@@ -35,6 +35,38 @@ check_neres_env() {
     check_neres_bin
 }
 
+get_k8s_shortname() {
```

Comment:
That's probably useful but not used in this PR. Just a heads up in case you didn't mean to include this